flood_api/settings.py

```diff
@@ -7,7 +7,13 @@ class Settings(BaseSettings):
     uvicorn_host: str = "0.0.0.0"
     uvicorn_reload: bool = True
     uvicorn_proxy_headers: bool = False
-    database_url: str = "postgresql+psycopg://postgres:postgres@localhost:5432/postgres"
+    flood_data_path_detailed: str = (
+        "s3://openepi-public-data/processed_detailed_forecast.parquet/"
```

Comment:
Should this also be overridden by an environment variable perhaps?

Reply:
I was considering that too, but I thought this was a convenient default for the moment.

Would you want me to hardcode the S3-paths in the k8s-YAML instead? I'm not sure if I see what we gain by moving it there.

A better option would perhaps be to deploy the data-bucket in IaC, and then create a ConfigMap that could be referenced in the k8s-YAML. The fact that we only have one databricks environment, but several environments for the rest of the platform makes this a bit more complicated though